internal/schema/stacks/1.9/output_block.go

```diff
@@ -48,6 +48,11 @@ func outputBlockSchema() *schema.BlockSchema {
 					IsRequired:  true,
 					Description: lang.PlainText("Type of the output value"),
 				},
+				"ephemeral": {
```

Comment:
I think we should also add the `sensitive` attribute here, as it goes hand in hand with ephemeral.

Reply:
added!